plugins/repository-hdfs/src/test/resources/rest-api-spec/test/hdfs_repository/20_repository.yaml

```diff
@@ -0,0 +1,25 @@
+# Integration tests for HDFS Repository plugin
+#
+# Check plugin is installed
+#
+"HDFS Repository Config":
+    - do:
+        snapshot.create_repository:
+          repository: test_repo_hdfs_1
+          verify: false
+          body:
+            type: hdfs
+            settings:
+              # local HDFS implementation
+              conf.fs.es-hdfs.impl: "org.elasticsearch.repositories.hdfs.TestingFs"
```

Comment:
As I've mentioned, the reason the `FS` is in there (and yes, it is a real HDFS file-system and a valid one as one can use a local API) is to overcome some of the current build limitations.
It's easy to remove it it and start an over the network HDFS (which will run locally just the same) assuming the build allows it.

Having a test in place, though not perfect, is better in my opinion than having no test as all.